test/java/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelTest.java

```diff
@@ -49,7 +49,7 @@ protected void onSetUp() {
   @Test
   @Ignore
   public void export_dir_button_opens_a_file_dialog() {
-    component = ConfigurationPanelPageObject.setUpDefaultPanel(robot(), empty().build(), true, true);
```

Comment:
I lost track of changes :D. 

Adjust as needed depending on whether the setUpDefaultPanel has to lose the arg or not.